Makefile

```diff
@@ -29,11 +30,18 @@ LOADER_OBJS = $(OUT)/loader.o $(OUT)/test_loader.o
 $(OUT)/loader: $(LOADER_OBJS)
 	$(CC) -o $@ $(LOADER_OBJS)
 
-$(CHECK_CC_ARCH):
-	@echo "Check cross compiler exist or not"
-	@echo "CROSS_COMPILE_SUFFIX=$(CROSS_COMPILE_SUFFIX)"
+$(CHECK_CC_ARCH)::
+	@echo "Check cross compiler CROSS_COMPILE_SUFFIX=$(CROSS_COMPILE_SUFFIX) exist or not"
+	@echo "If failed, please specify CROSS_COMPILE_SUFFIX"
 	@which $(patsubst check_cc_%,%,$@)$(CROSS_COMPILE_SUFFIX)-gcc
-	@echo "Pass"
+
+check_cc_aarch64::
+	@$(eval LD_VERSION=$(shell echo `aarch64$(CROSS_COMPILE_SUFFIX)-ld -v | grep -oE '[^ ]+$$'`))
+	@$(eval LD_VERSION=$(shell echo $(LD_VERSION) | awk -F "." '{print $$1$$2}'))
+	@if [ $(LD_VERSION) -lt 228 ]; then \
```

Comment:
Explain why you have to validate GNU linker versions here.

Reply:
Inline text describing the above considerations should be placed.

Reply:
Sorry I'm not really understand.
Do you mean
```
# explain why we check this.
#
check_cc_aarch64::
@$(eval LD_VERSION=$(shell echo `aarch64$(CROSS_COMPILE_SUFFIX)-ld -v | grep -oE '[^ ]+$$'`))
 +	@$(eval LD_VERSION=$(shell echo $(LD_VERSION) | awk -F "." '{print $$1$$2}'))
echo "Error: aarch64$(CROSS_COMPILE_SUFFIX)-ld version must >= 2.28 in AARCH64."; \
 +		echo "\tThe old version ld will corrupt global variable with -shared option."; \
 +		echo "\tFor example, func_table will contain NULL after linking test_lib.so."; \
```
OR
```
# explain why we check this.
#
check_cc_aarch64::
@$(eval LD_VERSION=$(shell echo `aarch64$(CROSS_COMPILE_SUFFIX)-ld -v | grep -oE '[^ ]+$$'`))
 +	@$(eval LD_VERSION=$(shell echo $(LD_VERSION) | awk -F "." '{print $$1$$2}'))
echo "Error: aarch64$(CROSS_COMPILE_SUFFIX)-ld version must >= 2.28 in AARCH64."; \
```

Reply:
I meant the later. Instead of mentioning in Git commit messages, the reason why LD versions should be validated is expected to fill into the area before target `check_cc_aarch64`.

Reply:
Okay, how about this version?